examples/predictors/tests/test_xprize_predictor.py

```diff
@@ -5,12 +5,12 @@
 
 import pandas as pd
 
-from ..xprize_predictor import XPrizePredictor
+from examples.predictors.lstm.xprize_predictor import XPrizePredictor
```

Comment:
I moved `tests` to be its own thing since these do not depend on the LSTM predictor. They only "borrow" the copy of the Oxford data file which "should" end up under `lstm/data`. But the alternative is to commit a copy of the Oxford file (eww) or download it live for the test (eww eww) so I'm not sure what else to do.

Reply:
Should be fixed now thanks to https://github.com/leaf-ai/covid-xprize/pull/91